pkg/service/v0/service.go

```diff
@@ -56,9 +72,12 @@ func NewService(opts ...Option) Service {
 
 // WopiServer defines implements the business logic for Service.
 type WopiServer struct {
-	logger log.Logger
-	config *config.Config
-	mux    *chi.Mux
+	serviceID string
+	logger    log.Logger
+	config    *config.Config
+	mux       *chi.Mux
+	c         *http.Client
```

Comment:
it might add more meaning renaming `c` to `httpClient` and `cs3Client` to `client`. This is an idiom we follow elsewhere within the codebase: https://github.com/cs3org/reva/search?q=gateway.GatewayAPIClient (see the function signatures)